docroot/themes/contrib/civictheme/theme-settings.php

```diff
@@ -7,17 +7,26 @@
 
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Link;
-use Drupal\Core\StreamWrapper\PublicStream;
 use Drupal\Core\StreamWrapper\StreamWrapperManager;
 use Drupal\Core\Url;
 
 /**
  * Implements hook_form_system_theme_settings_alter().
  */
 function civictheme_form_system_theme_settings_alter(&$form, &$form_state) {
-  $theme_name = \Drupal::configFactory()->get('system.theme')->get('default');
-  $theme_path = \Drupal::service('extension.list.theme')->getPath($theme_name);
+  civitheme_form_system_theme_settings_theme_version($form);
```

Comment:
please revert. this was done this way to adhere to how Drupal theme settings forms are usually built